Makefile

```
@@ -495,7 +495,10 @@ uninstall:
 		"$(DEST)/bin/kubectl.lima" \
 		"$(DEST)/share/man/man1/lima.1" \
 		"$(DEST)/share/man/man1/limactl"*".1" \
-		"$(DEST)/share/lima" "$(DEST)/share/doc/lima"
+		"$(DEST)/share/lima" "$(DEST)/share/doc/lima" \
```

I know you are just appending the final backslash here, but I think having multiple entries on one line is bad, and it should be split up. Sorry for not noticing in my preview review:

```suggestion
		"$(DEST)/share/lima" \
		"$(DEST)/share/doc/lima" \
```